src/components/link.js

```diff
@@ -199,7 +205,7 @@ module.exports.Component = registerComponent('link', {
       cameraWorldPosition.setFromMatrixPosition(camera.matrixWorld);
       distance = elWorldPosition.distanceTo(cameraWorldPosition);
 
-      if (distance > 20) {
+      if (distance > this.previewDistance + 5) {
```

Comment:
Why `+ 5` and not making previewDistance `25`?

Reply:
I left the functionality unchanged, when the scale is unchanged.  

The original code in master shows the preview when the camera is closer than 15m, and turns the link to face the camera when the camera is farther than 20m (you could call that the "link-lookAt distance'). In between, the center of the link just shows red, but the link isn't turned to face the camera.

I vary the preview distance, rather than the link-lookAt distance, because neither should be zero or less.  Also, the preview distance is used in several places in the code, and link-lookAt distance only one.

Any change in the relationship between these parameters or change to the behavior of links with scale 1, should be done in a different PR, I think.